frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -615,14 +615,6 @@ const ManageSoftwarePage = ({
       queryParams.vulnerable === "true");
 
   const renderSoftwareTable = () => {
-    if (
-      isFetchingCount ||
-      isFetchingSoftware ||
-      !globalConfig ||
-      (!softwareConfig && !softwareConfigError)
-    ) {
-      return <Spinner />;
```

Comment:
Root issue: can't use a spinner to replace the table level or else it will re-render (and thus unfocus) the search input field